CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit
+helps, and credit will always be given. You can contribute in the ways listed below.
+
+## Report Bugs
+
+Report bugs using GitHub issues.
+
+If you are reporting a bug, please include:
+
+* Your operating system name and version.
+* Any details about your local setup that might be helpful in troubleshooting.
+* Detailed steps to reproduce the bug.
+
+## Fix Bugs and Implement Features
+
+Look through the GitHub issues for bugs and features. Anything tagged with "bug", "help
+wanted" or "enhancement" is open to whoever wants to implement it.
+
+## Write Documentation
+
+The ORCESTRA book could always use more documentation, whether through describing a
+a sub-campaign or an instrument measurement or by adding a data processing example.
+Please feel free to add information yourself or open an issue if you lack the time 
+or knowledge to add it right away.
+
+
+## Get Started
+
+Ready to contribute? Here's how to set up the `ORCESTRA book` for local development.
+
+1. Fork the repo on GitHub.
+2. Clone your fork locally.
+3. Install your local copy into a virtualenv, e.g., using `conda`.
```

Comment:
We could maybe get into more details here, e.g. something like
```
conda env create -f environment.yml
```
should work to set up an `orcestra_book` environment.

Reply:
yes. I'll add that.
And it's correct to recommend a workflow with forks? I'd think so, but wanted to check once again.

Reply:
Yes, I guess we want a fork-based workflow.